### ISSUE TITLE
MarlinUtil HelixClass should not be used

### ISSUE THREAD

**Issue by tmadlener** [key4hep/k4RecTracker#31]
I just discovered this via #29. The `HelixClass` from MarlinUtil has quite a few quirks and is ideally superseded by something a bit more sane and with fewer rough edges. For an extended discussion see [MarlinUtil#24](https://github.com/iLCSoft/MarlinUtil/issues/24)

**Comment by BrieucF** [comment]
Is there anything replacing this already?

**Comment by tmadlener** [comment]
No, not really unfortunately. I think it is easiest to wait for ACTS and see whether we can reuse something from there. The main reason for opening this issue was to make people aware of the potential issues with the `HelixClass`. Ideally, we probably want to have some sort of abstraction here that acts as a kind of interface, but doesn't expose the details of what is underneath, such that we can "simply replace" that if necessary.
